apps/music/test/marionette/Player_test.js

```diff
@@ -133,4 +133,39 @@ marionette('Music player tests', function() {
       music.checkPlayerIconShown(true);
     });
   });
+
+  suite('Rating test', function() {
+    test('Rating persistance', function() {
+      music.launch();
+      music.waitForFirstTile();
+      music.switchToSongsView();
+      music.playFirstSong();
+
+      var rating_value = 4;
+
+      music.tapRating(rating_value);
+
+      // wait that the rating bar disappear.
+      client.waitFor(function() {
+        return !client.findElement(Music.Selector.ratingBar).displayed();
+      });
+
+      // tap to make the rating bar reappear.
+      client.helper.waitForElement(Music.Selector.coverImage).click();
+
+      // find all the stars that are on.
+      var stars = client.findElements('button.star-on');
+      assert.equal(stars.length, rating_value);
```

Comment:
Maybe add a getRating() utility function to music.js along with tapRating? A function that returns the number of highlighted stars seems like it would be useful.